package.json

```
@@ -40,7 +40,7 @@
     "path-to-regexp": "^6.2.1",
     "postman-collection": "^4.1.4",
     "reflect-metadata": "^0.1.13",
-    "@apidevtools/swagger-parser": "^10.1.0",
+    "swagger-parser": "^10.0.3",
```

```suggestion
    "@apidevtools/swagger-parser": "10.0.3",
```

`swagger-parser` is just a redirect to `@apidevtools/swagger-parser`, so I think it's better to directly depend on the latter.  The only reason `swagger-parser@^10.0.3` even works, is because the maintainers didn't publish `swagger-parser@10.1.0` when they published `@apidevtools/swagger-parser@10.1.0`.  I'm not sure if this was intentional or a mistake.

If you depend on `swagger-parser@^10.0.3`, and the maintainers do publish a new version of `swagger-parser` in the future, you will be broken again.

---

Thanks for reminding, I'll lock @apidevtools/swagger-parser to v10.0.3